### DISCUSSION TITLE
about normal color fg

### DISCUSSION THREAD

**Discussion by Archdsenna** [sxyazi/yazi#1286] [1. Q&A]
Hi, I tried to set a custom theme color, but I found that the background of the Normal label will change. When I open it from the terminal, the color of NORMAL is like this
![image](https://github.com/user-attachments/assets/0ecc994e-9ad6-425b-8184-7fe293b8a86e)
Then I selected a file from the list in yazi to open it, and when I closed the file and returned to the yazi interface again, the color of NORMAL changed to the following:
![image](https://github.com/user-attachments/assets/42fd368c-280b-46bc-95f9-18c4519c491f)
It looks like fg has changed. I tried to modify it in the theme.toml file, but I don’t know where to set this value. Can anyone help me?Thanks!

**Comment by xfzv** [comment]
> Hi, I tried to set a custom theme color, but I found that the *background* of the Normal label will change

You mean *foreground*?

> It looks like fg has changed. I tried to modify it in the theme.toml file, but I don’t know where to set this value.

Should be

```toml
# Mode
mode_normal = { fg = "value", bg = "value", bold = true }
mode_select = { fg = "value", bg = "value", bold = true }
mode_unset = { fg = "value", bg = "value", bold = true }
```

Maybe you took the two screenshots while experimenting with `theme.toml` and therefore using different colors, but there are quite a lot of other colors differences in your two screenshots:
- `?` icon on the left of "NORMAL" looks dimmer in the second screenshot(not sure why you have this by the way, also appear twice at the bottom right of the status bar)
- size foreground looks dimmer in the first screenshot
- currently selected file icon looks dimmer in the first screenshot
- currently selected file foreground is greyish on the first screenshot, black on the second one
- permissions foreground looks dimmer in the first screenshot
- total number of directories/files (`/4`) looks dimmer in the first screenshot
- `yazi.toml` icon looks dimmer in second screenshot
- Separators (vertical `|`) look dimmer in the second screenshot 

Are you getting these issues with a predefined flavor such as [catppuccin-mocha.yazi](https://github.com/yazi-rs/flavors/tree/main/catppuccin-mocha.yazi) for instance?

**Reply by Archdsenna** [reply]
Thank you for your reply, I just use theme.toml as the theme file, not flavor.Those strange characters are symbols that I did not modify the dividing line

Yes, I modified the color values ​​of mode_normal, such as fg and bg. This is the configuration related to these contents

`[flavor]
use = ""
`
`[status]
mode_normal = { fg = "231", bg = "23", bold = true }`

I did some more testing and found that this is yazi's behavior and has nothing to do with the configuration file. This phenomenon is the same as what I mentioned at the beginning. When you enter yazi from the command line, the color of fg is displayed normally, the same as set in theme.toml (for mode_normal), then select a file in the yazi directory list to open it, then close the file and return to yazi, you will find that the color of fg has become brighter
![20240713090014_rec_](https://github.com/user-attachments/assets/77c6f0ee-0731-42ca-b792-1e37b4ba00d2)

**Reply by Archdsenna** [reply]
Here is a clearer comparison
![image](https://github.com/user-attachments/assets/cd02e2be-9e41-4cca-9487-a29c19e763ff)

**Comment by Archdsenna** [comment]
Maybe the Mac terminal does not support some of the yazi usage, which is normal on iterm2. [Here ](https://github.com/sxyazi/yazi/issues/1289#event-13495312512)is the answer to this question. I closed this session.
